doc/classes/Node2D.xml

```diff
@@ -101,7 +101,7 @@
 			<argument index="0" name="global_point" type="Vector2">
 			</argument>
 			<description>
-				Converts a global point's coordinates to local coordinates.
+				Transforms the provided global position into a position in local coordinate space. The output will be local relative to the [Node2D] it is called on. i.e. It is appropriate for determining the positions of child nodes, but it is not appropriate for determining its own position.
```

Comment:
"..determining its own position relative to its parent" Maybe? Might help clarify for some users. It can be really difficult to make coordinate spaces easy to understand.

Reply:
Good idea, added that on.